pyshelf/search/update_manager.py

```diff
@@ -0,0 +1,73 @@
+from pyshelf.search.metadata import Metadata
+
+
+class UpdateManager(object):
```

Comment:
As discussed, add a function that I can pass a list of ids to and everything not in that list will get removed.